src/startunit.F90

```diff
@@ -638,9 +649,69 @@ subroutine InitializeProject(iproject, TheProjectFile, TheProjectType)
+                call fProjects_write(trim(tempstring))
+            end if
+            if (.not. FileOK%Temperature_Filename) then
+                write(tempstring, '(4a)') '               Temperature (Tnx of TMP), '
+                call fProjects_write(trim(tempstring))
+            end if
+            if (.not. FileOK%ETo_Filename) then
+                write(tempstring, '(4a)') '               Reference ET (ETo), '
+                call fProjects_write(trim(tempstring))
+            end if
+            if (.not. FileOK%Rain_Filename) then
+                write(tempstring, '(4a)') '               Rainfall (PLU), '
+                call fProjects_write(trim(tempstring))
+            end if
+            if (.not. FileOK%CO2_Filename) then
+                write(tempstring, '(4a)') '               CO2 (CO2), '
+                call fProjects_write(trim(tempstring))
+            end if
+            if (.not. FileOK%Calendar_Filename) then
+                write(tempstring, '(4a)') '               Calendar (CAL), '
+                call fProjects_write(trim(tempstring))
+            end if
+            if (.not. FileOK%Crop_Filename) then
+                write(tempstring, '(4a)') '               Crop (CRO), '
+                call fProjects_write(trim(tempstring))
+            end if
+            if (.not. FileOK%Irrigation_Filename) then
+                write(tempstring, '(4a)') '               Irrigation (Irr), '
+                call fProjects_write(trim(tempstring))
+            end if
+            if (.not. FileOK%Management_Filename) then
+                write(tempstring, '(4a)') '               Field Management (MAN), '
+                call fProjects_write(trim(tempstring))
+            end if
+            if (.not. FileOK%GroundWater_Filename) then
+                write(tempstring, '(4a)') '               Soil profile (SOL), '
+                call fProjects_write(trim(tempstring))
+            end if
+            if (.not. FileOK%Soil_Filename) then
+                write(tempstring, '(4a)') '               Groundwater (GWT), '
+                call fProjects_write(trim(tempstring))
+            end if
+            if (.not. FileOK%SWCIni_Filename) then
+                write(tempstring, '(4a)') '               Initial conditions (SW0), '
+                call fProjects_write(trim(tempstring))
+            end if
+            if (.not. FileOK%OffSeason_Filename) then
+                write(tempstring, '(4a)') '               Off-season (OFF), '
+                call fProjects_write(trim(tempstring))
+            end if
+            if (.not. FileOK%Observations_Filename) then
+                write(tempstring, '(4a)') '               Field data (OBS), '
+                call fProjects_write(trim(tempstring))
+            end if
+            write(tempstring, '(4a)') '          - Check file Name(s), Path(s) &
+                               or Structure of project file.'
```

Comment:
Was this check also tested in Fortran? E.g. run with a missing/erroneous path to see if it returns the right message.

Reply:
Yes, I tested my implementation. I added now additionally a print to screen information that this information on missing files is written into OUTP/ListProjectsLoaded.OUT

Reply:
Great!